setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = alfasim_sdk
+description = ALFAsim API/SDK
+url = https://github.com/esss/alfasim-sdk
+version = 0.1.0
```

Comment:
This should be removed to avoid confusion, because we let `setuptools-scm` handle versionining.